src/pages/gen2/build-a-backend/add-aws-services/custom-resources/index.mdx

```diff
@@ -50,16 +80,16 @@ The publisher Lambda allows publishing a message containing the email subject, b
 The `CustomNotifications` custom CDK construct can be defined as follows:
 
 ```ts title="amplify/custom/CustomNotifications/resource.ts"
-import { Runtime } from "aws-cdk-lib/aws-lambda";
+import { Runtime } from 'aws-cdk-lib/aws-lambda';
 import {
   NodejsFunction,
-  NodejsFunctionProps,
-} from "aws-cdk-lib/aws-lambda-nodejs";
-import * as sns from "aws-cdk-lib/aws-sns";
-import * as subs from "aws-cdk-lib/aws-sns-subscriptions";
-import { Construct } from "constructs";
-import { dirname, join } from "path";
-import { fileURLToPath } from "url";
+  NodejsFunctionProps
+} from 'aws-cdk-lib/aws-lambda-nodejs';
```

Comment:
All of these formatting changes were done automatically. At first glance it seems like these changes are consistent with the style in other code samples